cf-agent/verify_files_utils.c

```diff
@@ -3243,14 +3244,22 @@ static void SaveSetxid(bool modified)
     {
         char filename[CF_BUFSIZE];
         snprintf(filename, CF_BUFSIZE, "%s/cfagent.%s.log", GetLogDir(), VSYSNAME.nodename);
+        ToLowerStrInplace(filename);
         MapName(filename);
 
         PurgeItemList(&VSETXIDLIST, "SETUID/SETGID");
 
         Item *current = RawLoadItemList(filename);
         if (!ListsCompare(VSETXIDLIST, current))
         {
+            mode_t oldmode = umask(022);
+            Log(LOG_LEVEL_DEBUG, "Updating setxidlist at '%s', current umask is %o", filename, oldmode);
+            if(safe_chmod(filename, 0600))
```

Comment:
Space after `if`.

Reply:
removed code, not needed since RawSaveItemList() creates a new file each time anyway.